llvm/lib/Target/ARM/ARMInstrFormats.td

```diff
@@ -155,7 +155,12 @@ def iflags_op : Operand<i32> {
 
 // ARM Predicate operand. Default to 14 = always (AL). Second part is CC
 // register whose default is 0 (no register).
-def CondCodeOperand : AsmOperandClass { let Name = "CondCode"; }
+def CondCodeOperand : AsmOperandClass {
+  let Name = "CondCode";
+  let PredicateMethod = "isCondCode";
```

Comment:
Looks redundant (the name of the method should be correctly inferred).

Reply:
Done